server/portal/settings/unit_test_settings.py

```diff
@@ -280,7 +280,12 @@ def gettext(s): return s
 PORTAL_PROJECTS_PUBLIC_KEY = 'ssh-rsa change this'
 
 PORTAL_USER_ACCOUNT_SETUP_STEPS = [
-    'portal.apps.onboarding.steps.test_steps.MockStep'
+    {
```

Comment:
after looking at `abstract.py`,  i wonder if a unique name is needed for each step? 

as the settings are currently found by finding the first matching `step` value.   right now, we could have steps that use the same python class, right?

Reply:
We could, but I think explicit is more desirable than implicit. I also have a file test_steps with multiple classes inside of it. If we had other situations like that, we could have file explosion.